EMML/7001-8000/EMML7942.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+        type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="EMML7942" type="mss">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title xml:id="t1">EMML 7942</title>
+            <editor role="generalEditor" key="AB"/>
+            <funder>Akademie der Wissenschaften in Hamburg</funder>
+         </titleStmt>
+         <editionStmt>
+            <p>The initial version of this file was created from data kindly
+                                provided by Hill Museum &amp; Manuscript Library.</p>
+            <p>Permalink: <ref type="mss" target="https://w3id.org/vhmml/readingRoom/view/201315">https://w3id.org/vhmml/readingRoom/view/201315.</ref>
+            </p>
+            <p/>
+         </editionStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <pubPlace>Hamburg</pubPlace>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas:  Eine
+                                multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                  <p>This XML file is licensed under the Creative Commons
+                                        Attribution-ShareAlike 4.0. </p>
+               </licence>
+            </availability>
+            <date>2017-11-30+01:00</date>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+               <msIdentifier>
+                  <repository ref="INS0004HMML"/>
+                  <collection>, </collection>
+                  <idno>MS EMML no. 7942</idno>
+               </msIdentifier>
+               <msContents>
+                  <msItem xml:id="ms_i1">
```

Comment:
Note that this description is wrong, as the manuscripts contain multiple texts (sets of OT books, at least all the Minor Prophets), one of which is Enoch